manifests/0000_05_config-operator_02_config.clusterrole.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: "true"
+  creationTimestamp: null
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-cluster-reader: "true"
+  name: system:openshift:aggregate-to-cluster-reader
```

Comment:
sorry, just noticed this.  `system:openshift:config.openshift.io-access`

Reply:
I don't see an .io-access in the file. Does it need to be added?